webuni/commonmark-bundle/0.2/etc/packages/webuni_commonmark.yaml

```diff
@@ -0,0 +1,6 @@
+webuni_commonmark:
+    extensions:
+        # adds support for defining attributes on HTML elements (e.g. 'id', 'class', 'style')
+        attributes: true
+        # adds support for parsing Markdown tables
+        table: true
```

Comment:
this repeats the default values.

Thus, it does not always enable it: setting it to true here will have an effect only if the extension is installed in your project (which btw is a weird behavior. It would be better to change the default value based on installation status and then enforce to have the extension when enabling it. Much less WTF)